hw/ip_templates/otp_ctrl/dv/env/seq_lib/otp_ctrl_base_vseq.sv.tpl

```diff
@@ -60,6 +60,15 @@ class otp_ctrl_base_vseq extends cip_base_vseq #(
 
   otp_ctrl_callback_vseq callback_vseq;
 
+  // Some sequences have specific ISR to manage interrupt state registers and respond asynchronously
+  // to interrupts generated by the OTP
+  bit isr_present = 0;
+
+  // When read from a zeroized locked location is attempted, dai_idle check fails as the RTL
```

Comment:
Seems like another TODO, since reads from locked partitions (regardless of zeroization) will also return immediately. Rather than block the idle check could we predict it will never go non-idle?